server/src/serializers/ReviewSerializer.js

```diff
@@ -4,13 +4,15 @@ class ReviewsSerializer {
   static async getReviewDetail(review) {
     const user = await review.$relatedQuery("user");
     const allowedAttributes = ["rating", "description", "id"];
-
+    const dog = await review.$relatedQuery("dog")
     let serializedReview = {};
     for (const attribute of allowedAttributes) {
       serializedReview[attribute] = review[attribute];
     }
 
     serializedReview.userName = user.name;
+    serializedReview.dogName = dog.dogName
+    serializedReview.dogId = dog.id
```

Comment:
Why not attach the whole dog above when it's queried?
```js
serialiedReview.dog = await review.$relatedQuery("dog")
```

Reply:
We figured then we would be grabbing all the reviews for the dog as well when we have no use for the data.